app/src/forms/email/emailService.js

```diff
@@ -96,41 +156,51 @@ const service = {
         bodyTemplate: 'send-status-assigned-email-body.html',
         title: `${form.name} Submission Assignment`,
         subject: 'Form Submission Assignment',
-        messageLinkText: 'You have been assigned to review this submission.',
+        messageLinkText: `You have been assigned to a ${form.name} submission. Please login to review it.`,
         priority: 'normal',
         assignmentNotificationEmail,
+        form,
       };
       return service._sendEmailTemplate('sendStatusAssigned', configData, submission, referer);
     } catch (e) {
-      log.error(`status: ${currentStatus}, referer: ${referer}`, { function: 'statusAssigned' });
-      log.error(e.message, { function: 'statusAssigned' });
-      log.error(e);
+      log.error(e.message, e, {
+        function: 'statusAssigned',
+        status: currentStatus,
+        referer: referer
+      });
       throw e;
     }
   },
 
-  submissionConfirmation: async (formId, submissionId, body, referer) => {
+  // Submitting Form
+  submissionReceived: async (formId, submissionId, body, referer) => {
     try {
       const form = await formService.readForm(formId);
       const submission = await formService.readSubmission(submissionId);
       const configData = {
         bodyTemplate: 'submission-confirmation.html',
         title: `${form.name} Submission`,
         subject: `${form.name} Submission`,
+        messageLinkText: `There is a new ${form.name} submission. Please login to review it.`,
         priority: 'normal',
         body,
         form,
       };
-      return service._sendEmailTemplate('sendSubmissionConfirmation', configData, submission, referer);
+      return service._sendEmailTemplate('sendSubmissionReceived', configData, submission, referer);
     } catch (e) {
-      log.error(`formId: ${formId}, submissionId: ${submissionId}, body: ${JSON.stringify(body)}, referer: ${referer}`, { function: 'submissionConfirmation' });
-      log.error(e.message, { function: 'submissionConfirmation' });
-      log.error(e);
+      log.error(e.message, e, {
+        function: 'submissionReceived',
+        formId: formId,
+        submissionId: submissionId,
+        body: JSON.stringify(body),
+        referer: referer
+      });
       throw e;
     }
   },
 
-  submissionReceived: async (formId, submissionId, referer) => {
+  // Sending confirmation of Form Submission
+  submissionConfirmation: async (formId, submissionId, body, referer) => {
     try {
       const form = await formService.readForm(formId);
```

Comment:
Believe this should have the identity provider in it to figure out if the form is public or not (see review comment)

Reply:
doing a check for the identityProviders to remove the link on public.